Sophos/sophos firewall/_meta/smart-descriptions.json

```
@@ -160,5 +160,29 @@
         "type": "connection"
       }
     ]
+  },
+  {
+    "value": "Connection with {network.transport} protocol {action.name} between {source.ip} and {destination.ip}",
+    "conditions": [
+      {
+        "field": "network.transport"
+      },
+      {
+        "field": "destination.ip"
+      },
+      {
+        "field": "action.name"
+      },
+      {
+        "field": "source.ip"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "source.ip",
+        "target": "destination.ip",
+        "type": "connection"
```

To be consistent with [other](https://github.com/SEKOIA-IO/intake-formats/blob/a228ce01f44a7a8485a90849c5b2dba4034600a2/SkyhighSecurity/skyhigh_secure_web_gateway/_meta/smart-descriptions.json#L153) [formats](https://github.com/SEKOIA-IO/intake-formats/blob/a228ce01f44a7a8485a90849c5b2dba4034600a2/Imperva/imperva-waf/_meta/smart-descriptions.json#L30):
```suggestion
        "type": "connected to"
```